homework/01LessonOne.js

```diff
@@ -153,47 +154,47 @@ var notNotUndefined;
 
```

Comment:
Make sure to include an answer to this question:
 Explain truthiness and falsiness in your own words. Provide an example for each.